ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -454,6 +454,71 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws-upi,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e-aws-upi-proxy
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    name: pull-ci-openshift-installer-master-e2e-aws-upi-proxy
+    optional: true
+    rerun_command: /test e2e-aws-upi-proxy
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-aws-upi-proxy-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-aws-upi-proxy
+        - --template=/usr/local/e2e-aws-upi-proxy
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-installer-master.yaml
+              name: ci-operator-master-configs
+        - name: JOB_NAME_SAFE
+          value: e2e-aws-upi-proxy
+        - name: TEST_COMMAND
+          value: TEST_SUITE=openshift/conformance/parallel run-tests
```

Comment:
@wking how do I pass an env var to the template to be run? adding `USE_PROXY` here fails the prow check and recommends it be removed...

Reply:
templates take parameters, any env var you define here will be passed as a parameter to the template.

then the scripting inside the template can use the parameter name.

Reply:
(e.g. you should see that the template takes a TEST_COMMAND parameter and then uses it later to invoke the defined command)

Reply:
I had that, but the prow/generated-config job was flagging that added env var, and if I ran the recommended command to resolve it the env var would be removed... I'll poke around some more, maybe I need to update my local repo

Reply:
yeah the generator will stomp this file for the most part.  we need to see if there's a way to inject the var in the config file, not the job file, so the generator propagates it.  @stevekuznetsov might know if there's a way to do that today.

Reply:
Where is `USE_PROXY` from?  I'd have expected you to set `PROXY_ENABLED` in this `env` list.

Reply:
:man_facepalming: 
yeah that was the wrong env var name... not sure why `USE_PROXY` got in my head as the one that was being used

Reply:
It still isn't a fan of additional env vars being added [1]
[1] https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/4591/pull-ci-openshift-release-master-generated-config/7511